### PR TITLE
Ci/watch hot reload

### PR DESCRIPTION
**Pull request by iBat**

### CODE REVIEW

File: packages/react/project.json [hunk 1]
```diff
@@ -8,7 +8,6 @@
             "executor": "nx:run-commands",
             "options": {
                 "commands": [
-                    "tsc -b -w ./tsconfig.package.json",
```
Comment (williamvinogradov) [inline packages/react/project.json:11]: If I remember correctly, the only reason why the React package has a project.json is that we need a complicated command for React here.
We simplified the command for the React package.

Maybe we can delete project.json for this package now?

File: playgrounds/react/package.json [hunk 2]
```diff
@@ -29,6 +29,7 @@
     "build": "react-scripts build",
     "start": "react-scripts start",
     "serve": "npm start",
```
Comment (williamvinogradov) [inline playgrounds/react/package.json]: Let's remove the duplicate npm command

```suggestion
```

File: playgrounds/angular/package.json [hunk 3]
```diff
@@ -3,9 +3,10 @@
   "version": "1.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
+    "start": "nx serve @playgrounds/angular",
     "serve": "npm start",
```
Comment (williamvinogradov) [inline playgrounds/angular/package.json]: Let's remove the duplicate npm command

```suggestion
```

File: playgrounds/angular/project.json [hunk 4]
```diff
@@ -0,0 +1,83 @@
+{
```
Comment (williamvinogradov) [inline playgrounds/angular/project.json:1]: Minor commment / Quiestion.

Why is the angular config file new? 
What happened with the old version?

Comment (iBat) [reply playgrounds/angular/project.json:1]: With the old version, the application was not updated when the libraries changed.

**Review by williamvinogradov — APPROVED** [review]

**Review by MikeVitik — CHANGES_REQUESTED** [review]
Add jquery watch

**Review by alexslavr — APPROVED** [review]

File: tsconfig.base.json [hunk 5]
```diff
@@ -19,7 +19,7 @@
     "noPropertyAccessFromIndexSignature": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "noEmitOnError": true,
+    "noEmitOnError": false,
```
Comment (MikeVitik) [inline tsconfig.base.json]: Move this noEmitOnError into the watch rollup task only

**Review by MikeVitik — CHANGES_REQUESTED** [review]

**Review by MikeVitik — APPROVED** [review]

**Review by const314 — APPROVED** [review]

**Pull request merged**